src/components/ChattingScreen.js

```diff
@@ -18,14 +21,96 @@ export default function ChattingScreen() {
 		{ user: false, content: '나도 아니까  ' },
 		{ user: false, content: '그만 말해줘도 돼' },
 		{ user: true, content: 'ㅠㅠ' },
-	];
+	]);
+	function changeSender(e){
+		e.preventDefault();
+		setCurrentSender(!currentSender);
+	}
+	function changeNewMsg(e){
+		e.preventDefault();
+		setNewMsg({user: currentSender, content: e.target.value});
+	}
+	function updateMsglist(e){
+		e.preventDefault();
+		if(newMsg.content === ''){
+			alert('메시지를 입력해주세요');
+			return;
+		}
+		setCurrentMsglist([...currentMsglist, newMsg]);
+		setNewMsg({user: currentSender, content: ''});
+	}
+	let currentSenderName = currentSender ? USER_T_NAME : USER_F_NAME;
+	let currentSenderIMG = currentSender ? USER_T_IMG : USER_F_IMG;
+
+	useEffect(() => {
+		window.scrollBy(0, window.innerHeight);
+	}, [currentMsglist]);
 
 	return (
 		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
+			<Header sender = {currentSenderName} senderImg = {currentSenderIMG} changeSender = {changeSender} ></Header>
+			{currentMsglist.map((message) =>
+				<PrintOneMessage sender = {message.user} message = {message.content}/>
```

Comment:
컴포넌트의 이름은 명사형으로 지어 주는 것이 좋습니다 ㅎㅎ

```suggestion
				<Message sender = {message.user} message = {message.content}/>
```

Reply:
감사합니다 훨씬 낫네요!